src/sentry/conf/server.py

```diff
@@ -761,22 +761,26 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
 CELERYBEAT_SCHEDULE = {
     "check-auth": {
         "task": "sentry.tasks.check_auth",
-        "schedule": timedelta(minutes=1),
+        # Run every 1 minute
+        "schedule": crontab(minute="*/1"),
```

Comment:
I would give a heads up on #team-engineering before merging this early in the day and be around in case any of this backfires. Just being paranoid.